src/components/item/FolderContent.tsx

```diff
@@ -49,6 +50,10 @@ const Content = ({ item, searchText, items, sortBy }: Props) => {
   const { itemTypes } = useFilterItemsContext();
   const { selectedIds, clearSelection, toggleSelection } =
     useSelectionContext();
+  const theme = useTheme();
+  const DragSelection = useDragSelection({
+    adjustments: { marginTop: 70, marginLeft: theme.spacing(3) },
```

Comment:
Where are these defined ? can we use the same constants ?

Reply:
These depends on the layout.. so it's kind of unstable, but it depends on many params unfortunately =/